x/evm/keeper/grpc_query.go

```diff
@@ -664,6 +664,7 @@ func (k *Keeper) traceTx(
 	// reset gas meter for tx
 	// to be consistent with tx execution gas meter
 	ctx = ctx.WithGasMeter(evmostypes.NewInfiniteGasMeterWithLimit(msg.Gas()))
+	ctx = ctx.WithKVGasConfig(storetypes.GasConfig{})
```

Comment:
Need set zero for transient gas config also

Reply:
Need to set on `TraceTx` also

Reply:
Request back port v16 for Escan resyncing data

Reply:
We will backport this and create a patch release @VictorTrustyDev

Reply:
Hey @VictorTrustyDev! There is no need to set it on `TraceTx` because this calls `traceTx` internally. We only want this gas configuration with ETH transactions. 

Added the `transientGasConfig` as well, great catch!